src/addresses.ts

```diff
@@ -143,6 +145,12 @@ export const ADDRESSES: {
     1: '0x80DC468671316E50D4E9023D3db38D3105c1C146',
     42: '0xb36f06dB2F9EE10879640C53E3E09Ee4C476597D',
   },
+  [ALPHA]: {
+    1: '0xa1faa113cbe53436df28ff0aee54275c13b40975',
```

Comment:
Address always needs to be in checksum format
```suggestion
    1: '0xa1faa113cbE53436Df28FF0aEe54275c13B40975',
```